apps/dashboard/test/models/motd_formatter/motd_formatter_plaintext_erb_test.rb

```diff
@@ -1,55 +1,41 @@
 require 'test_helper'
 
 class MotdFormatter::PlaintextErbTest < ActiveSupport::TestCase
-  include MotdFormatter
   test "plaintext-formatter-erb returns a valid motd file when given a valid motd file" do
     path = "#{Rails.root}/test/fixtures/files/motd_valid"
-    motd_file = MotdFile.new(path)
-    formatted_motd = MotdFormatterPlaintextErb.new(motd_file)
-    expected_file = File.open(path).read
+    with_modified_env({ 'MOTD_FORMAT': "text_erb", 'MOTD_PATH': path }) do
```

Comment:
The initial test was plain text format. Indeed this class name also reinforces that - `PlaintextErbTest`.